VideoWeb/VideoWeb/ClientApp/package-lock.json

```diff
@@ -4096,8 +4096,6 @@
             "resolved": "https://registry.npmjs.org/ajv/-/ajv-8.6.3.tgz",
```

Comment:
can you uncommit this change, just to be on the safe side, unless you are entirely sure this wouldnt cause any issue, i always try not to include changes in this file

Reply:
done